bitnami/mongodb/templates/standalone/svc.yaml

```diff
@@ -27,18 +24,14 @@ spec:
   loadBalancerSourceRanges: {{- toYaml .Values.service.loadBalancerSourceRanges | nindent 4 }}
   {{- end }}
   ports:
-    - name: mongodb
+    - name: tcp-mongodb
```

Comment:
Hey @juan131 , we just installed mongo using version 8.0.7 of the mongodb chart and noticed that we aren't able to connect to the deployed mongo using this connection string: `mongodb+srv://mongodb-headless.persistence.svc.cluster.local`

I found out that the reason is, that the port within the `mongodb-headless` service is named `tcp-mongodb` instead of `mongodb` and then I found this change here.

Could you please have a look again at your changes and ensure that connecting to mongo via `srv` still works?
This also affects the other service definitions that you created during your refactorings.

More information about this: 
* https://www.mongodb.com/blog/post/mongodb-3-6-here-to-SRV-you-with-easier-replica-set-connections -> mongo prepends `_mongodb._tcp` to the hostname 
* https://kubernetes.io/docs/concepts/services-networking/service/#dns: k8s is using the port name + protocol in the SRV DNS entry as a prefix

Reply:
Hi,

Thank you so much for the feedback. Could create an issue instead?  This will likely get lost if we use PR comments. I can copy-paste the message in a new issue if necessary.

Reply:
Good idea, I just created #3067 out of this discussion